tests/unit/amo/pages/TestUsersUnsubscribe.js

```diff
@@ -115,22 +98,22 @@ describe(__filename, () => {
         token: params.token,
       }),
     );
-    const dispatchSpy = sinon.spy(store, 'dispatch');
+    const dispatch = jest.spyOn(store, 'dispatch');
 
     render({ params, store });
 
-    sinon.assert.notCalled(dispatchSpy);
+    expect(dispatch).not.toHaveBeenCalled();
```

Comment:
I am surprised this is passing. `dispatch` always seems to be called at least once when we call `render`, but maybe that's because you are rendering a component and not a page. You will likely find that when you convert this to use `renderPage` this assertion starts failing, in which case you'll need to assert that `dispatch` was not called with an action with the type for `unsubscribeNotification` (i.e., via `expect.objectContaining({ type: THE_ACTION_TYPE })`).